naucse/templates/session.html

```diff
@@ -0,0 +1,38 @@
+{% extends "_base.html" %}
+
+{% block content %}
+
+<div class="page">
+    <div class="container">
+
+        {% if run is defined %}
+            <header class="lesson-header">
+                <h1>{{ run.title }}</h1>
+                <hr>
+            </header>
+        {% endif %}
+
+        {{ content }}
+
+        <hr class="lesson-end">
+
+        {% if (prv is defined and prv) or (nxt is defined and nxt) %}
+        <div class="row prev-next">
+            {% if prv %}
+                <div class="col text-left">
+                    <a href="{{ lesson_url(prv.lesson, page=prv.slug) }}">← {{ prv.title }}</a>
+                </div>
+            {% endif %}
+
+            {% if nxt %}
+                <div class="col text-right">
+                    <a href="{{ lesson_url(nxt.lesson, page=nxt.slug) }}">{{ nxt.title }} →</a>
+                </div>
+            {% endif %}
+        </div>
+        {% endif %}
```

Comment:
This should be made into a macro. Could you start a template file with utility macros? [Docs are here.](http://jinja.pocoo.org/docs/2.9/templates/#import)